test/_external/test262/skiplist

```diff
@@ -99,3 +105,33 @@ test/language/module-code/instn-star-props-nrml.js
 
 # NEEDS INVESTIGATION!
 test/language/module-code/parse-err-reference.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/define-own-property.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/delete-exported-init.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/delete-exported-uninit.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/delete-non-exported.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/get-own-property-str-found-init.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/get-str-found-init.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/get-str-initialize.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/get-str-update.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/own-property-keys-sort.js
+
+# NEEDS INVESTIGATION!
+test/language/module-code/namespace/internals/set.js
```

Comment:
We throw here too but be careful as by default `esm` enables CJS interop with mutable namespace objects (which don't throw). This is configurable though `cjs.mutableNamespace`.